SampleApplications/Workshop/DataTypes/_docs/IntegrateOwnInformationModel.md

```diff
@@ -0,0 +1,79 @@
+﻿# How to integrate new information model into OPC Server
+
+This documentation explains how to add a custom information model to OPC Server based on UA-.NETStandard stack. It will use the DataTypes server example as reference but the general steps are the same for every UA-.NETStandard stack based OPC Server.
+
+## Preparation
+
+1. Clone [Opc.Ua.ModelCompiler Repository](https://github.com/OPCFoundation/UA-ModelCompiler)
+2. Build Opc.Ua.ModelCompiler solution
+3. Copy the build result of Opc.Ua.ModelCompiler solution to [UA-.NETStandard/SampleApplications/bin](./../../bin)
+
+## Add own information model
+
+1. Create a Folder to to UA-.NETStandard/SampleApplications/Workshop/DataTypes/Common e.g. MyInformationModel
+   1. Create a sub-folder named "Output"
+2. Copy the model itself into this folder e.g. MyInformationModel.xml into UA-.NETStandard/SampleApplications/Workshop/DataTypes/Common/MyInformationModel
+3. Modify [BuildDesign.bat](./Common/BuildDesign.bat) and add the following lines
+
```

Comment:
one small nitpick, since adding a model to a library is rather uncommon I'd describe the process as if the new model is added to the actual server. Hopefully just some path changes ..

Reply:
I really liked the design of DataTypes server, to add the model into a library, so it will be possible use the library on client aswell on server (and for testing). 

if you say it is uncommon I can update the documentation - should I copy the BuildDesign.bat also to Server or link it from the current location?